shared/xccdf/system/auditing.xml

```diff
@@ -896,7 +896,7 @@ cui="3.3.1" />
 
 <Rule id="file_ownership_var_log_audit" severity="medium" prodtype="rhel7">
 <title>System Audit Logs Must Be Owned By Root</title>
-<description>
+<description>Checks that all /var/log/audit files and directories are owned by the root user and group.
 <fileowner-desc-macro file="/var/log" owner="root"/>
```

Comment:
@redhatrises I truly don't know :disappointed: 

I'm not familiar with <fileowner-desc-macro>, do you think it should be changed to that?